### PR TITLE
version.rc: Add xemu branding to windows

### PR DESCRIPTION
**Pull request by MasonT8198**

### CODE REVIEW

**Comment by MasonT8198** [comment]
This should fix issue #222

File: version.rc [hunk 1]
```diff
@@ -13,12 +13,12 @@ FILESUBTYPE VFT2_UNKNOWN
   {
     BLOCK "040904E4"
     {
-      VALUE "CompanyName", "https://www.qemu.org"
-      VALUE "FileDescription", "QEMU machine emulators and tools"
+      VALUE "CompanyName", "https://www.xemu.app"
+      VALUE "FileDescription", "XEMU Xbox Emulator"
       VALUE "FileVersion", QEMU_VERSION
       VALUE "LegalCopyright", "Copyright various authors. Released under the GNU General Public License."
       VALUE "LegalTrademarks", "QEMU is a trademark of Fabrice Bellard."
```
Comment (mzji) [inline version.rc:28]: Maybe this `QEMU` can also be replaced?

File: version.rc [hunk 2]
```diff
@@ -13,12 +13,12 @@ FILESUBTYPE VFT2_UNKNOWN
...
     {
-      VALUE "CompanyName", "https://www.qemu.org"
-      VALUE "FileDescription", "QEMU machine emulators and tools"
-      VALUE "FileVersion", QEMU_VERSION
-      VALUE "LegalCopyright", "Copyright various authors. Released under the GNU General Public License."
-      VALUE "LegalTrademarks", "QEMU is a trademark of Fabrice Bellard."
-      VALUE "ProductName", "QEMU"
+      VALUE "CompanyName", "https://www.xemu.app"
+      VALUE "FileDescription", "XEMU Xbox Emulator"
+      VALUE "FileVersion", XEMU_VERSION
```
Comment (mzji) [inline version.rc]: `QEMU_VERSION` is actually a predefined macro, thus it cannot be replaced so simply...

Comment (MasonT8198) [reply version.rc]: Sorry about that, changed the wrong one 🥲

File: version.rc [hunk 3]
```diff
@@ -13,12 +13,12 @@ FILESUBTYPE VFT2_UNKNOWN
...
-      VALUE "CompanyName", "https://www.qemu.org"
-      VALUE "FileDescription", "QEMU machine emulators and tools"
+      VALUE "CompanyName", "https://www.xemu.app"
+      VALUE "FileDescription", "XEMU Xbox Emulator"
       VALUE "FileVersion", QEMU_VERSION
-      VALUE "LegalCopyright", "Copyright various authors. Released under the GNU General Public License."
-      VALUE "LegalTrademarks", "QEMU is a trademark of Fabrice Bellard."
-      VALUE "ProductName", "QEMU"
+      VALUE "LegalCopyright", "Copyright Matt Borgerson 2020-2021. Released under the GNU General Public License."
+      VALUE "LegalTrademarks", "XEMU is a trademark of Matt Borgerson."
```
Comment (dracc) [inline version.rc]: Is this actually true?
Also I'm not sure it's a good idea to alter the qemu copyright stuff.

Comment (MasonT8198) [reply version.rc]: Sorry about that, I’ll revert it back to qemu, since it’s built on qemu

File: version.rc [hunk 4]
```diff
@@ -13,12 +13,12 @@ FILESUBTYPE VFT2_UNKNOWN
   {
     BLOCK "040904E4"
     {
-      VALUE "CompanyName", "https://www.qemu.org"
-      VALUE "FileDescription", "QEMU machine emulators and tools"
+      VALUE "CompanyName", "https://www.xemu.app"
+      VALUE "FileDescription", "XEMU Xbox Emulator"
```
Comment (mborgerson) [inline version.rc]: xemu: Original Xbox Emulator

File: version.rc [hunk 5]
```diff
@@ -13,12 +13,12 @@ FILESUBTYPE VFT2_UNKNOWN
...
       VALUE "FileVersion", QEMU_VERSION
-      VALUE "LegalCopyright", "Copyright various authors. Released under the GNU General Public License."
+      VALUE "LegalCopyright", "Copyright Various Authors. Released under the GNU General Public License."
```
Comment (mborgerson) [inline version.rc]: Capitalization change not necessary

File: version.rc [hunk 6]
```diff
@@ -13,12 +13,12 @@ FILESUBTYPE VFT2_UNKNOWN
   {
     BLOCK "040904E4"
     {
-      VALUE "CompanyName", "https://www.qemu.org"
-      VALUE "FileDescription", "QEMU machine emulators and tools"
+      VALUE "CompanyName", "https://www.xemu.app"
+      VALUE "FileDescription", "XEMU Xbox Emulator"
       VALUE "FileVersion", QEMU_VERSION
```
Comment (mborgerson) [inline version.rc:26]: Make sure this actually has the xemu version in it

File: version.rc [hunk 7]
```diff
@@ -13,12 +13,12 @@ FILESUBTYPE VFT2_UNKNOWN
...
-      VALUE "LegalCopyright", "Copyright various authors. Released under the GNU General Public License."
+      VALUE "LegalCopyright", "Copyright Various Authors. Released under the GNU General Public License."
       VALUE "LegalTrademarks", "QEMU is a trademark of Fabrice Bellard."
-      VALUE "ProductName", "QEMU"
+      VALUE "ProductName", "XEMU"
```
Comment (mborgerson) [inline version.rc]: Lowercase `xemu` is what I've been using

**Review by mborgerson — CHANGES_REQUESTED** [review]

**Pull request merged**